### PR TITLE
WIP: Team leading articles

### PR DESCRIPTION
**Pull request by saddy4**
The following is the omnicomprehensive template for the Pull Request.

Complete the section about your specific scenario.

### 1. Submit a new article

_Follow this template to check if your article can be submitted._

Please check to satisfy these requirements:

* [X] Please check your author information: `/_data/authors.yml`: if it is not present add it.
* [x] In the article meta the `tags` section is used for the team name. The team list can be find under the section `themeColor` in `_config.yml`: if your team is not present just add it.
* [x] Double check the grammar using Grammarly: https://www.grammarly.com/ (using American English).
* [X] If you can, reduce the code - you're explaining a concept, hence having lots of code makes the article only longer without adding a real information. Short code examples -> detailed description of the concept is the way to go.
* [X] Keep the article formal - reduce/remove the slang expressions and the punctuations like `...` if not strictly necessary.
* [X] Use the oxford comma (setting up Grammarly to use American English should forces you to use it)
* [x] Do not hotlink images: put the images inside a subfolder in the images folder of the blog and link there. Double check the images copyright before using them.
* [X] The ideal article should start from a problem -> the article content should start from it and describe the flow to solve it.

### CODE REVIEW

**Comment by saddy4** [comment]
**Missing**

- [x] I put a table in one article but it seems to be not supported. I'm not sure it will stay in the article anyway, I need to check it with the UX team
- [x] I don't know if we need different Pull request to make the article appears gradually, I was planning to release them one or two a week.
- [x] Still missing images for the article, I will find them with the UX team.

File: _posts/2020-08-03-making-a-leader-part-3.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+description: "A quick overview on how do we choose and train leaders at zuru"
+---
+
+<a href="/images/teamleading3.svg"><img class="blog-image" style="width: 100%" src="/images/teamleading3.svg"> </a>
+
+Welcome to the third article in the series! In this article, I will focus on how each skill can be useful to boost other skills.
+
+In the image below, you will see every skill we talked about until now, and how they can merge to support each other or how they originate new skills!
+
+TODO IMAGE
```
Comment (galeone) [inline _posts/2020-08-03-making-a-leader-part-3.md]: ??

File: _posts/2020-08-03-making-a-leader-part-3.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+## Communication to the next level
```
Comment (galeone) [inline _posts/2020-08-03-making-a-leader-part-3.md:22]: remove initial whitespace and add newline

File: _posts/2020-08-03-making-a-leader-part-3.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+In the previous articles, I talked about how communication is essential to leading for various reasons. However, there is one thing that can make the difference between a good communicator and an excellent communicator: the ability to motivate others.
+Everyone can be a good leader by being a good communicator without motivating others, and it is okay. However, if you can make them passionate about what they do, this can make the difference.
+If you can motivate your teams, with words and by examples, they will be more responsible, passionate, and less stressed.
+
+All the next skills need good communication ability; without it, they are almost useful.
+
+ ## Problem-Solving + Managing skills = Organizational skills
```
Comment (galeone) [inline _posts/2020-08-03-making-a-leader-part-3.md]: initial whitespace

File: _posts/2020-08-03-making-a-leader-part-3.md (hunk 1 above)
Comment (saddy4) [reply _posts/2020-08-03-making-a-leader-part-3.md]: Fixed right now!

**Comment by saddy4** [comment]
@galeone it should be ready. In the local preview, anyway i get this error with the preview coloro or the text boxes, i don't know why. I add the category 
image

**Comment by galeone** [comment]
@saddy4 I guess is somehow related with the `did` field of the articles, if I remember correctly for some reason they all should be different. Try

**Pull request merged**